src/java.compiler/share/classes/javax/lang/model/util/Types.java

```diff
@@ -35,7 +35,19 @@
 /**
  * Utility methods for operating on types.
  *
- * Where a method returns a type mirror or a collection of type
+ * Most methods operate on {@linkplain PrimitiveType primitive types},
+ * {@linkplain ReferenceType reference types} (including {@linkplain
+ * ArrayType array types} and the {@linkplain NullType null type}),
+ * {@linkplain IntersectionType intersection types}, and the
+ * pseudo-type '{@link TypeKind#VOID void}'. {@linkplain
+ * ExecutableType Executable types} and the pseudo-types for
+ * {@linkplain TypeKind#PACKAGE packages} and {@linkplain
+ * TypeKind#MODULE modules} are generally out of scope for these
+ * methods. One or more out of scope arguments will typically result
+ * in a method throwing an {@link IllegalArgumentException
+ * IllegalArgumentException}.
```

Comment:
Nit: do you need the second argument? It should be the default string.

Reply:
I'll check if "java.lang" is inserted without the second argument.